src/main/java/com/cloudbees/jenkins/support/impl/SlaveCommandStatistics.java

```diff
@@ -143,29 +154,43 @@ long sum() {
 
         private final Set<File> jars = new LinkedHashSet<>();
 
+        private static void preferringOlder(Map<String, CountSum> map, String key, long value) {
```

Comment:
I see Codahale Metrics already in use in this project, perhaps we can apply it in here as well.